File: src/main/java/aero/sort/vizualizer/data/options/styles/concrete/Sunrise.java
// -------------------------------------------------------------------------------
// Copyright (c) Ladendorfer Daniel.  
// All Rights Reserved.  See LICENSE in the project root for license information.
// -------------------------------------------------------------------------------
package aero.sort.vizualizer.data.options.styles.concrete;

import aero.sort.vizualizer.data.options.styles.AbstractGradientStyle;
import aero.sort.vizualizer.ui.constants.Theme;

import java.awt.*;

/**
 * Sunrise gradient style.
 *
 * @author Daniel Ladendorfer
 */
public class Sunrise extends AbstractGradientStyle {
    @Override
    protected Color getPrimaryColor() {
        return Theme.CYAN;
    }

    @Override
    protected Color getSecondaryColor() {
        return Theme.YELLOW;
    }
}

File: src/main/java/aero/sort/vizualizer/data/options/styles/concrete/gradient/Sunrise.kt
// -------------------------------------------------------------------------------
// Copyright (c) Ladendorfer Daniel.  
// All Rights Reserved.  See LICENSE in the project root for license information.
// -------------------------------------------------------------------------------
package aero.sort.vizualizer.data.options.styles.concrete.gradient

import aero.sort.vizualizer.data.options.styles.AbstractGradientStyle
import aero.sort.vizualizer.ui.constants.Theme
import java.awt.Color

/**
 * Sunrise gradient style.
 *
 * @author Daniel Ladendorfer
 */
class Sunrise : AbstractGradientStyle() {
    override fun getPrimaryColor(): Color = Theme.CYAN
    override fun getSecondaryColor(): Color = Theme.YELLOW
}